lib/converter.rb

```diff
@@ -42,6 +42,26 @@ class Converter
     _: '',
   }
 
+  # Taken from:
+  # https://github.com/perseids-project/morphsvc/blob/master/morphsvc/lib/transformers/LatinTransformer.py
+  LATIN_TRANSFORMS = [
+    [/[^\w\d\s]/, ''],
```

Comment:
Like the comment notes, this is taken from the `LatinTransformer.py` file in `morphsvc`. An interesting side effect of applying these to only the Latin input is that Latin words with punctuation work but Greek words with punctuation don't work.

For example: [Greek](https://morph.perseids.org/analysis/word?lang=grc&engine=morpheusgrc&word=%CF%80%CF%8C%CE%BD%CF%84%E1%BF%B3?) and [Latin](https://morph.perseids.org/analysis/word?lang=lat&engine=morpheuslat&word=fero?).

Reply:
that's interesting. I see that in the Alpheios version too, and can't understand why it would be.

Reply:
Looking at the regex (`r'[^\w|\d|\s]'`) again this morning, I realized that the Alpheios version does not remove the `|` character. I assume this is a bug and the regex and it should be `r'[^\w\d\s]'`.

Reply:
ah, yes that does appear to be a bug in the regex -- the | should not be there. Thanks!

Reply:
so now it makes sense to me about the punctuation

Reply:
Made a pull request here: https://github.com/alpheios-project/morphsvc/pull/37